usm: Added documentation for bare metal support (#18681)
DataDog/documentation: content/en/universal_service_monitoring/_index.md

Required Agent version
: Universal Service Monitoring requires that the Datadog Agent installed alongside your containerized service be at least version 6.40 or 7.40.

> For non-containerized services see instructions [here](7)

Reviewer: I think you meant to put this below under Prerequisites :)

Author: great catch
moved it

- If on Linux:
    - Your service is running in a container.

>     - For non-containerized services see instructions [here][7]

Reviewer 2: This is the format we use for anchor links (though your way also works in my local build, not sure how it is translated in the build)
```suggestion
    - For non-containerized services see the [instructions here](#support-for-non-containerized-services).
```

{{< /tabs >}}

> ### Non containerized support

Reviewer 2: ```suggestion
### Non-containerized support
```

Reviewer: I'd suggest we change the title to: `Support for non-containerized services (Public Beta)`

Reviewer: Lets also move this section to be immediately after **Exploring your services**

Author: @sunhay, the linter does not allow having Public Beta in the headline (capitalization issues)
is it enough to have the "public beta" message here?
![image](https://github.com/DataDog/documentation/assets/17148247/d49f53e6-2ce5-4b79-92b5-c99158dc4e60)

Reviewer: I think that should be fine! @estherk15 any thoughts?

Reviewer 2: Yes! This looks great!

> Universal Service Monitoring available in *beta* to monitor services running bare-metal on Linux virtual machines.

Reviewer 2: ```suggestion
<div class="alert alert-info">
Universal Service Monitoring is available in *beta* to monitor services running bare-metal on Linux virtual machines.
</div>
```

[4]: /monitors/types/apm/?tab=apmmetrics
[5]: /dashboards/
[6]: /service_management/service_level_objectives/metric/

> [7]: #support-for-non-containerized-services

Reviewer 2: ```suggestion
```
